docker/build_scripts/build_utils.sh

```diff
@@ -0,0 +1,101 @@
+#!/bin/bash
+# Helper utilities for build
+
+PYTHON_DOWNLOAD_URL=https://www.python.org/ftp/python
+OPENSSL_DOWNLOAD_URL=http://www.openssl.org/source
+GET_PIP_URL=https://bootstrap.pypa.io/get-pip.py
+
+
+function check_var {
+    if [ -z "$1" ]; then
+        echo "required variable not defined"
+        exit 1
+    fi
+}
+
+
+function pyver_ge {
+    # Echo 1 if first python version is greater or equal to second
```

Comment:
Sorry, over-trusting copy from stackoverflow :-) echo doesn't interpret escape sequences, so it should be:
```
major_minor_version_lte() {
  printf "$1\n$2" | sort -n -t "." -k 1,1 -k 2,2 -s | head -n 1 | grep -q "$1"
}
```
which gives
```
[root@84345d3a5606 /]# major_minor_version_lte 3.3 3.5; echo $?
0
[root@84345d3a5606 /]# major_minor_version_lte 3.5 3.3; echo $?
1
```